src/auth/__mocks__/auth.controller.mock.ts

```diff
@@ -19,6 +19,7 @@ export class MockAuthController {
       id: faker.number.int(),
       email: signUpRequest.email,
       password: signUpRequest.password,
+      enabled2FA: false,
```

Comment:
this name kind of doesn't sit with me - maybe `secondFactorEnabled` or `mfaEnabled`. Something else so that the `enabled` part is at the end